src/lv_conf_internal.h

```diff
@@ -530,6 +530,17 @@ e.g. "stm32f769xx.h" or "stm32f429xx.h"*/
 #  endif
 #endif
 
+#else
+#  define LV_LOG_LEVEL            LV_LOG_LEVEL_NONE
```

Comment:
lv_conf_internal.h is generated by python script, you shouldn't modify this file manually.

Reply:
It's a little bit odd, but it seems easier to fix it in the source files. E.g.
```c
#if LV_USE_LOG && LV_LOG_TRACE_EVENT
    #define EVENT_TRACE(...) LV_LOG_TRACE(__VA_ARGS__)
#else
    #define EVENT_TRACE(...)
#endif
```

Reply:
I've modified the script to parse a new LVGL specific pragma in the template so that defines will not be expanded. That lets the header ensure conditional defines have a default value.

Reply:
If we add it to `lv_conf_template.h` the `lv_conf.h` of the user will be noisier and longer. 

If you don't like the suggestion in my [previous comment](https://github.com/lvgl/lvgl/pull/2780/files#r745719207) and another option could be to manually add the problematic cases at the bottom of `lv_conf_internal.h` (i.e. generate via the script).

What do you think?